aten/src/ATen/native/cuda/BatchLinearAlgebra.cu

```diff
@@ -1601,6 +1643,46 @@ Tensor _cholesky_helper_cuda(const Tensor& self, bool upper) {
   }
 }
 
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ cholesky_inverse ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+/*
+Computes the inverse of a symmetric (Hermitian) positive-definite matrix n-by-n matrix 'input' using the Cholesky factorization
+This is an in-place routine, content of 'input' is overwritten.
+'infos' is an int Tensors containing error codes for each matrix in the batched input.
+For more information see MAGMA's documentation for POTRI routine.
+*/
+template <typename scalar_t>
+static void apply_cholesky_inverse(Tensor& input, Tensor& infos, bool upper) {
+#ifndef USE_MAGMA
+  AT_ERROR("cholesky_inverse: MAGMA library not found in "
+      "compilation. Please rebuild with MAGMA.");
+#else
+  magma_uplo_t uplo = upper ? MagmaUpper : MagmaLower;
+
+  auto input_data = input.data_ptr<scalar_t>();
+  magma_int_t n = magma_int_cast(input.size(-2), "input.size(-2)");
+  auto lda = std::max<magma_int_t>(1, n);
+
+  if (input.dim() == 2) {
+    TORCH_INTERNAL_ASSERT(infos.device() == at::kCPU);
+    magmaCholeskyInverse<scalar_t>(uplo, n, input_data, lda, infos.data_ptr<magma_int_t>());
+  } else {
+    TORCH_CHECK(false, "Batched version is not implemented for CUDA.")
```

Comment:
"Batched version is not implemented for CUDA." -> "cholesky_inverse does not support batched inputs on CUDA"

We want error messages to include the user-facing name of the function that throws them.

Reply:
Thanks, I'll change it. To be clear `apply_cholesky_inverse` does not support batched input, but `cholesky_inverse` supports it via a call to `cholesky_solve`, so this error should never be thrown, but it's safer to keep it here.